### PR TITLE
Fix Dockerfile and README for etcd/flannel

### PR DESCRIPTION
**Pull request by yuqi-zhang**
In response to comments from bugzilla, add following changes:

1. Pin container to specific fedora version
2. Chain labels
3. Add/fix README
4. Install packages without docs

Signed-off-by: Yu Qi Zhang <jerzhang@redhat.com>

### CODE REVIEW

**Comment by yuqi-zhang** [comment]
@giuseppe PTAL

File: etcd/Dockerfile [hunk 1]
```diff
@@ -8,9 +8,9 @@ LABEL BZComponent="etcd-docker" \
       Version="$VERSION" \
       Release="$RELEASE" \
       Architecture="$ARCH" \
-      Summary="A key-value store for shared configuration and service discovery, intended to be run as a system container"
+      Summary="A key-value store for shared configuration and service discovery."
 
-RUN dnf -y install etcd hostname && \
+RUN dnf -y --setopt=tsflags=nodocs install etcd hostname && \
```
Comment (ashcrow) [inline etcd/Dockerfile]: :+1:

File: flannel/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,8 @@
-FROM fedora
+FROM fedora:25
```
Comment (ashcrow) [inline flannel/Dockerfile]: :+1:

File: flannel/Dockerfile [hunk 3]
```diff
@@ -15,7 +13,7 @@ LABEL BZComponent="flannel-docker" \
       Summary="An etcd driven address agent, intended to be run as a system container" \
       atomic.type='system'
 
-RUN dnf -y install flannel && dnf clean all
+RUN dnf -y --setopt=tsflags=nodocs install flannel && dnf clean all
```
Comment (ashcrow) [inline flannel/Dockerfile]: :+1:

**Comment by yuqi-zhang** [comment]
Also added a PR to move maintainer into a label, in correspondence to: https://github.com/projectatomic/ContainerApplicationGenericLabels/issues/78

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 1787e3e

**Pull request closed without merging**